examples/app-basic/tests/appConfig.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { test, expect } from './fixtures';
+
+test('configuring app', async ({ appConfigPage, page }) => {
+  await appConfigPage.goto();
+
+  const saveForm = page.getByRole('button', { name: /Save API settings/i });
+  await expect(saveForm).toBeVisible();
+
+  // reset the configured secret
+  await page.getByRole('button', { name: /reset/i }).click();
+
+  // enter some valid values
+  await page.getByLabel('API Key').fill('secret-api-key');
+  await page.getByLabel('API Url').clear();
+  await page.getByLabel('API Url').fill('http://www.my-awsome-grafana-app.com/api');
```

Comment:
Nit, but would be good to interact with these form elements using the approach that we suggest in the [docs](https://grafana.com/developers/plugin-tools/e2e-test-a-plugin/selecting-elements#input-field). Should be `getByRole` for inputs.